packages/intl-durationformat/test262-main.ts

```diff
@@ -0,0 +1,1157 @@
+// @generated
+// @ts-nocheck
+import '@formatjs/intl-getcanonicallocales/polyfill'
+import '@formatjs/intl-locale/polyfill'
+import '@formatjs/intl-enumerator/polyfill'
+
+import '@formatjs/intl-listformat/polyfill-force'
```

Comment:
DurationFormat relies on polyfilled changes to ListFormat and NumberFormat... I didn't see the other test262 loading polyfills so if there is another way to approach this I can alter this.